doc/Rules.md

```diff
@@ -426,7 +426,7 @@ Aliases: no-reversed-links
 
 Fixable: Most violations can be fixed by tooling
 
-This rule is triggered when text that appears to be a link is encountered, but
+This rule is triggered when a piece of text that appears to be a link is encountered, but
```

Comment:
Again, I feel the original phrasing was fine, but happy to defer to @DavidAnson's judgment since this change is not incorrect, just redundant IMO.